### DISCUSSION TITLE
Add a hide/exclude button to each item in the history panel

### DISCUSSION THREAD

**Discussion by Mechotronic** [home-assistant/frontend#11237] [Other feature requests]
## The request

It would be nice to have something like an "X" to the right of each item that could be clicked to hide it. Even if it's just adding the exclude rule to configuration.yaml, it would save a lot of time not having to cross-reference all of that manually.

**Comment by zsarnett** [comment]
There is a plan to do this different with integration API that would expose the ability to mark as History or not in the UI

**Comment by GrizzlyAK** [comment]
I've entered another option as a feature request to allow wildcards in the Entity dropdown at the top, so I could say, view all entities that had `*temperature*` in either the `entity_id` or the `friendly_name`. Not exactly the same as this request, but there is enough overlap to keep in mind as this is implemented.

Although, it would also be nice to be able to separate the History from the Recorder (DB) so that you could filter the default History in the UI without having to modify config and restart HA.
